### ISSUE TITLE
KerasModel.compile() should support loss_weights

### ISSUE THREAD

**Issue by bzamecnik** [tensorpack/tensorpack#763]
Feature Request.

`keras.models.Model.compile(self, optimizer, loss=None, metrics=None, loss_weights=None, sample_weight_mode=None, weighted_metrics=None, target_tensors=None)`

while:

`tensorpack.contrib.keras.compile(self, optimizer, loss, metrics=None)`

**Comment by ppwwyyxx** [comment]
It doesn't seem reasonable to add more Keras support, so closing.
